source/_docs/timeouts.md

```diff
@@ -119,7 +119,7 @@ Yes, just use `terminus drush <site>.<env> -- cron` using [Terminus](/docs/termi
 
 As [strongly recommended by the Migrate module](https://www.drupal.org/node/1806824), use Drush, which can be invoked through [Terminus](/docs/terminus/). You can also configure Migrate to [trigger Drush imports from the UI](https://www.drupal.org/node/1958170) by configuring the `migrate_drush_path` variable to:
 ```
-$conf['migrate_drush_path'] = $_ENV['HOME'] . '/drush';
+$conf['migrate_drush_path'] = '/usr/local/bin/drush';

```

Comment:
This is going to be dependent on how the user installed Drush. The existing suggestion, however, is wrong all the time. The new suggestion (`/usr/local/bin/drush`) is very often correct, so it is a good improvement.

A note to confirm the installation path of Drush via `which drush` might be helpful.

Reply:
Oh wait, this is on Pantheon, not the local site. There's a better option, one moment.